app/actors/hyrax/actors/abstract_actor.rb

```diff
@@ -16,11 +16,11 @@ module Actors
     class AbstractActor
       attr_reader :next_actor
 
-      def initialize(_curation_concern, _user, next_actor)
+      def initialize(_curation_concern, _ability, next_actor)
```

Comment:
Wondering if we can have the following signature:

```ruby
def initialize(_curation_concern, user, next_actor, ability: default_ability)
  # insert setters here
  @next_actor = next_actor
  self.user = user
end

private

def default_ability
  Ability.new(@user)
end

def user=(input)
  # Deprecation notice saying to favor an ability
  @user = user
end
```

It preserves the current interface, but allows for an ability to be passed. I believe there would also be